.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,10 @@
+### Questions To Investigate
+
+Please explain the area that will be investigated during the spike with concrete
+questions that will answered.
```

Comment:
that will **be** answered

Reply:
Unbelievable! I was literally one word short of wordiness :).

Reply:
Explain the area to be investigated during the spike with concrete questions you want answered. 

(Just a rule of thumb in technical docs, don't use "please".)